### ISSUE TITLE
Multi-line props?

### ISSUE THREAD

**Issue by chris-martin** [sol/doctest#131]
I expected that the multi-line syntax

```
>>> :{
  ...
:}
```

would work for property test as well

```
prop> :{
  ...
:}
```

But this apparently doesn't parse.

Would it be possible to make this work? Or is there any other way to express property checks that span multiple lines?

**Comment by vimuel** [comment]
Yes, this is an absolute must-have! At the moment I'm having to declare the properties in the source file and then reference them in the comment. 👎
